src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -3,18 +3,150 @@
 using System.IO;
 using System.IO.Compression;
 using System.Linq;
+using System.Net.Http;
 using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.Extensions.Primitives;
+using Raven.Client.Exceptions.Database;
+using Raven.Client.Server;
 using Raven.Server.Documents.Handlers.Admin;
+using Raven.Server.Exceptions;
+using Raven.Server.Json;
 using Raven.Server.Routing;
 using Raven.Server.ServerWide;
 using Raven.Server.ServerWide.Context;
 using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Sparrow.Logging;
 
 namespace Raven.Server.Documents.Handlers.Debugging
 {
     public class ServerWideDebugInfoPackageHandler : AdminRequestHandler
     {
+        private static readonly string[] EmptyStringArray = new string[0];
+        private static readonly Logger _log = LoggingSource.Instance.GetLogger<ServerStore>(nameof(ServerWideDebugInfoPackageHandler));
```

Comment:
Logging here should be the other way around. The generic arg is the logging, and the name in this case is just the server.